cvat/scripts/pascal_voc.py

```diff
@@ -0,0 +1,42 @@
+import os
+import argparse
+import xml.etree.ElementTree
+from PIL import Image
+from pascal_voc_writer import Writer
+
+
+def process_cvat_xml(xml_file, img_dir, annotation_dir):
+    os.makedirs(annotation_dir)
+    cvat_xml = xml.etree.ElementTree.parse(xml_file)
+
+    for img_tag in cvat_xml.findall('image'):
+        filename = img_tag.get('name')
+
+        filepath = os.path.join(img_dir, filename)
+        with Image.open(filepath) as img:
+            width, height = img.size
+
+        writer = Writer(filepath, width, height)
+
+        for box in img_tag.findall('box'):
```

Comment:
Soon we are going to add polygons, polylines, points. What is your proposal? To convert them to bounding boxes or ignore?

Reply:
At first I would ignore and warn, after if we find some clever ways to convert those geometries into bounding boxes we could add a parameter for the user to choose (from the top of my head I guess an application of convex hull and obtain min/max coordinates would work), but in my opinion that should be another feature proposal